website/docs/configure_hydra/workdir.md

```diff
@@ -36,16 +31,91 @@ Run output directory can contain user configuration variables:
 hydra:
   run:
     dir: outputs/${now:%Y-%m-%d_%H-%M-%S}/opt:${optimizer.type}
+```
+
+### Configuration for multirun
 
+Sweep sub directory contains the job number for the job instance:
+```yaml
+hydra:
+  sweep:
+    dir: sweep_dir
+    subdir: ${hydra.job.num}
+```
```

Comment:
This is almost the default, except the dir does not have the timestamp.
So what you are actually doing here is to prevent the timestamp. Try to clarify what you are trying to demonstrate before adding examples.

Reply:
Before this yaml the doc says `Sweep sub directory contains the job number for the job instance. `

Are you saying we should add more context here?

Reply:
I am saying that you should make up your mind about what you are demonstrating.
In the code, you are overriding hydra.sweep.dir but keeping the subdir as the default.
It seems odd given that your sentence is talking about the sub dir.

Maybe open by describing the default behavior and then show a few ways to change it.

Reply:
got it! sorry I misunderstood you earlier.